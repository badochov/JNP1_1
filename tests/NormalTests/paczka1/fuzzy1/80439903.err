Error in line 6: Rtj6yS S119 13,8
Error in line 13: pyH A17 59,6
Error in line 15: S119 A17 51,5
Error in line 26: S119 A17 95,9
Error in line 22: eID3Wip A17 25,4
Error in line 32: S119 S119 2"3,8
Error in line 33: Rtj6yS A114 31,3
