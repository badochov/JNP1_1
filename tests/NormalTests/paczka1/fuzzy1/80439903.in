Lh4Aispt A114 8,2
Lh4Aispt A114 37,4
NCBFkd4 A17 75,1
?S119
?A114
Rtj6yS S119 13,8
Rtj6yS A17 81,5
NCBFkd4 A17 229,1

Rtj6yS A17 310,5
NCBFkd4 A17 38,1
NCBFkd4 A17 58,3
pyH A17 59,6
?
S119 A17 51,5
Lh4Aispt S979 18,5
pyH S119 93,1
S119 S979 20,6
pyH S119 154,5
S119 S979 90,6
?
eID3Wip A17 25,4
?
NCBFkd4 S119 81,6
?eID3Wip
S119 A17 95,9
S119 S119 45,5
Lh4Aispt S979 0,0
NCBFkd4 S119 88,1

eID3Wip S979 58,0
S119 S119 2"3,8
Rtj6yS A114 31,3
NCBFkd4 S979 48,3
eID3Wip S979 65,0
S119 S119 75,4
pyH A17 18,2
?eID3Wip
Rtj6yS S119 19,3
pyH A17 57,5