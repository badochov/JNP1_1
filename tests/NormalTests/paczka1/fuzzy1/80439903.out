A114 29,2
Lh4Aispt A 29,2
NCBFkd4 A 174,2
Rtj6yS A 229,0
A17 403,2
A114 29,2
Lh4Aispt A 29,2
NCBFkd4 A 174,2
Rtj6yS A 229,0
S119 S 70,0
pyH S 61,4
A17 403,2
A114 29,2
S119 61,4
S979 70,0
Lh4Aispt A 29,2
NCBFkd4 A 174,2
Rtj6yS A 229,0
S119 S 70,0
pyH S 61,4
A17 403,2
A114 29,2
S119 61,4
S979 70,0
eID3Wip S 7,0
